specification/netapp/resource-manager/Microsoft.NetApp/preview/2021-12-01-preview/examples/NetworkSiblingSet_Update.json

```diff
@@ -0,0 +1,49 @@
+{
+  "parameters": {
+    "subscriptionId": "D633CC2E-722B-4AE1-B636-BBD9E4C60ED9",
+    "location": "eastus",
+    "api-version": "2021-12-01-preview",
+    "body": {
+      "networkSiblingSetId": "9760acf5-4638-11e7-9bdb-020073ca3333",
+      "subnetId": "/subscriptions/9760acf5-4638-11e7-9bdb-020073ca7778/resourceGroups/myRP/providers/Microsoft.Network/virtualNetworks/testVnet/subnets/testSubnet",
```

Comment:
>ubscriptions/9760acf5-4638-11e7-9bdb-020073ca7778/resourceGroups/myRP/providers/Microsoft.Network/virtualNetworks/testVnet/subnets/testSubnet", [](http://example.com/codeflow?start=21&length=143)

Do you plan to add linked access checks to ensure that the called has appropriate access to this resource ?